apps/condo/domains/common/utils/mutations.utils.js

```diff
@@ -43,7 +43,7 @@ function runMutation ({ action, mutation, variables, onCompleted, onError, onFin
                 let notificationContext = null
                 if (ErrorToFormFieldMsgMapping) {
                     const errors = []
-                    const errorString = `${e}`
+                    const errorString = `${JSON.stringify(e.graphQLErrors)}`
```

Comment:
Will JSON.stringify display errors in a pretty way? These errors we are showing to users, not developers

Reply:
you need to check that error have `graphQLErrors` data

Reply:
> Will JSON.stringify display errors in a pretty way? These errors we are showing to users, not developers

it looks like `errorString` is not show to the user

Reply:
> Will JSON.stringify display errors in a pretty way? These errors we are showing to users, not developers

It is not shown to users. `errorString` is needed to find error keys passed to `ErrorToFormFieldMsgMapping` in it, then `ErrorToFormFieldMsgMapping` displays beautiful messages to users using these keys.

Reply:
> you need to check that error have `graphQLErrors` data

I think it's better to just do `JSON.stringify(e)` instead of `JSON.stringify(e.graphQLErrors)`